engine/src/main/java/org/terasology/telemetry/TelemetryUtils.java

```diff
@@ -47,30 +51,60 @@
      * @param bindingMap the binding map contains fields who has user's permission.
      */
     public static void trackMetric(Emitter emitter, String nameSpace, Unstructured event, Metric metric, Map<String, Boolean> bindingMap) {
-        Subject subject = new Subject.SubjectBuilder()
-                .userId(TelemetryParams.userId)
-                .build();
+        AccessController.doPrivileged((PrivilegedAction<Object>) () -> {
+            Subject subject = new Subject.SubjectBuilder()
+                    .userId(TelemetryParams.userId)
+                    .ipAddress("anonymous")
+                    .timezone("anonymous")
+                    .build();
 
-        // initialise tracker
-        Tracker tracker = new Tracker.TrackerBuilder(emitter, nameSpace, TelemetryParams.APP_ID_TERASOLOGY)
-                .subject(subject)
-                .platform(TelemetryParams.PLATFORM_DESKTOP)
-                .build();
+            // initialise tracker
```

Comment:
Thats what someone would expect when reading `Tracker tracker = new Tracker....`.
No extra information from the comment -> delete.